RockWeb/Blocks/Crm/BulkUpdate.ascx.cs

```diff
@@ -673,22 +697,40 @@ protected void btnComplete_Click( object sender, EventArgs e )
                 #region Tag
                 if ( !string.IsNullOrWhiteSpace( ddlTagList.SelectedValue ) )
                 {
-                    changes.Add( string.Format( "{0} {1} <span class='field-name'>{2}</span> tag.", 
+                    changes.Add( string.Format( "{0} {1} <span class='field-name'>{2}</span> tag.",
                         ddlTagAction.SelectedValue,
                         ddlTagAction.SelectedValue == "Add" ? "to" : "from",
                         ddlTagList.SelectedItem.Text ) );
                 }
                 #endregion
 
+                #region workflow
+
+                if ( !string.IsNullOrWhiteSpace( rlbWorkFlowType.SelectedValue ) )
+                {
+                    string wotkFlowTypes = string.Empty;
+                    foreach ( ListItem item in this.rlbWorkFlowType.Items )
+                    {
+                        if ( item.Selected )
+                        {
+                            wotkFlowTypes = wotkFlowTypes + item.Text + ",";
+                        }
+                    }
+                    changes.Add( string.Format( "Activate the Workflow <span class='field-name'>{0}</span>",
+                       wotkFlowTypes ) );
+                }
```

Comment:
Again, there are some extension methods that make delimiting list of strings (i.e. so comma is not after last item) and pluralizing text easier...
```
                    var wotkFlowTypes = new List<string>();
                    foreach ( ListItem item in this.rlbWorkFlowType.Items )
                    {
                        if ( item.Selected )
                        {
                            wotkFlowTypes.Add( item.Text );
                        }
                    }

                    changes.Add( string.Format( "Activate the <span class='field-name'>{0}</span> {1}.", 
                        wotkFlowTypes.AsDelimited(", ", " and " ), 
                        "workflow".PluralizeIf( wotkFlowTypes.Count > 1 ) ) );
```